src/components/OutboxMessageListItem.vue

```diff
@@ -176,22 +171,37 @@ export default {
 				failed: false,
 				sendAt: (new Date().getTime() + UNDO_DELAY) / 1000,
 			}
+
 			await this.$store.dispatch('outbox/updateMessage', { message, id: message.id })
 			try {
-				if (this.message.status !== STATUS_IMAP_SENT_MAILBOX_FAIL) {
-					await this.$store.dispatch('outbox/sendMessageWithUndo', { id: message.id })
-				} else {
-					await this.$store.dispatch('outbox/copyMessageToSentMailbox', { id: message.id })
+				await this.$store.dispatch('outbox/sendMessageWithUndo', { id: message.id })
+			} catch (error) {
+				logger.error('Could not send message', { error })
+				if (error.data !== undefined) {
+					await this.$store.dispatch('outbox/updateMessage', { message: error.data[0], id: message.id })
 				}
+			}
+		},
+		async copyToSentMailbox() {
```

Comment:
is this needed for the fix?

Reply:
I can revert but it simply splits the previous `if` into two separate methods